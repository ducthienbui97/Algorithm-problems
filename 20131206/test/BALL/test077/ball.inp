8 94916 99878 21536 378710
718653 542907 97926 17624 741429 494852 314697 801565 667652 892829 832240 475844 587662 358487 257398 399189 939777 550540 810143 194843 76988 937903 841330 340483 274342 970677 726001 663315 855705 553699 173149 455809 814810 625821 87670 799345 229426 239789 686970 721462 422344 249352 984874 728720 119928 532891 425455 191823 12728 235144 908304 268212 903630 51144 946747 14933 875745 977632 216934 435535 8297 693042 207307 122168 
290202 501093 380374 616212 637387 177217 804505 757993 774715 16555 398469 477876 664332 683540 781949 791294 976173 777419 918336 496051 733785 667182 364415 414377 629258 267764 742640 363585 750637 5162 892850 54965 753291 557703 99753 677287 35266 465025 981776 109918 132752 886187 933541 468283 823896 100201 583897 587316 906601 845326 333715 667231 12276 874277 339120 805281 380270 938481 946562 894519 128567 754231 841665 879028 
787861 865302 462852 120848 446631 672044 727656 228801 468339 89973 216375 70079 425192 328324 948963 124075 966714 47913 732669 281299 609225 68983 721004 459533 665502 704780 361988 437098 91081 20669 202328 875849 612785 411537 804778 404309 739019 292572 763229 426388 609535 512496 176677 506900 365234 472851 117666 341616 701199 826993 604101 48002 175089 405184 649886 278528 678999 584198 303968 456996 93758 72646 735461 314639 
957913 610369 454360 979989 478286 688834 203557 949302 33823 276799 774907 185425 271373 286371 205334 173755 945414 789689 861707 963672 366648 703489 815082 512672 930322 454808 570119 740623 761457 233253 96337 172127 337608 406580 815732 496605 141057 906025 239596 353108 930273 745715 739935 785284 746403 314615 33122 941527 581624 935485 249229 616603 366304 831875 205732 691131 193942 324188 747458 956825 554604 320560 439511 536426 672268 803050 694424 904721 841515 86725 78934 169711 883945 453573 413078 514515 103910 217993 2371 208518 429675 422746 93577 123717 903111 155261 581641 167952 840372 266235 966901 927118 566868 138796 429817 943077 905851 783201 960408 937508 918140 54597 412850 236965 263886 254754 800771 509958 247980 524030 496820 861572 401324 596258 381563 536416 233431 997260 457112 568526 121430 636985 443196 636721 620212 786004 230728 896059 738964 202370 18809 31145 517741 616825 70294 764202 587434 578925 312480 878853 364232 780591 224695 505044 998273 851319 870117 485233 160005 119656 588080 75563 988904 467713 89046 198391 781909 347772 624826 171535 835783 201097 574873 645490 950069 801058 955480 332098 180735 373444 647257 601309 221644 558966 786094 739173 613569 275828 60635 656423 864548 604519 379054 556260 726251 324922 219386 552331 251991 595580 339505 741141 655470 204235 667746 404226 710959 681469 52794 569399 197321 953291 60677 850539 783827 43015 419834 842694 204376 482009 840104 990220 701679 393589 236232 357917 516248 339424 68910 398456 759440 394870 393699 717931 786791 401763 757966 538861 961936 849343 125086 387751 859639 826249 919133 689275 623188 913902 806552 311691 878532 308379 542797 546746 78436 748299 937334 24157 302574 61144 744663 679519 475036 91748 743148 387670 56200 297614 301432 337235 299885 225663 504888 507028 772065 235520 938074 6221 172249 536571 623231 952906 636242 307646 740666 665380 345922 241737 206931 555193 466354 605917 43467 21622 402263 744064 250165 931264 56927 146885 743973 319013 378412 371412 563023 627548 270997 789913 661232 238796 492247 234789 95691 273176 414711 953212 906272 336186 488749 685288 891158 203541 39978 472491 277360 86225 502297 461069 180223 188418 259982 702833 348322 14262 353110 55860 680959 552008 733339 141533 749918 725500 264886 252456 1008 801377 463457 945244 509995 146101 510662 576809 312984 751067 319230 231951 968971 877283 469230 418148 535309 268937 736636 247318 968293 467836 499753 963614 736746 922178 372822 87484 912989 357858 490144 533176 786839 536246 749266 309784 154997 79711 461673 946610 952800 180424 421929 191386 461982 350190 916747 735240 912145 775523 955744 853041 486094 227446 559781 281397 810693 838479 45254 162927 25202 546563 164341 855347 680077 519834 527776 180762 209071 515185 642135 211123 50451 804553 165266 20049 285342 755321 26497 594876 440290 107449 885246 347238 4478 388796 407283 490493 172997 867681 898904 165789 550747 89320 860626 558946 768127 984397 753509 212291 508218 695011 133871 729026 653660 721348 91057 889479 728504 174338 494930 41016 741734 775577 252908 386455 763884 135390 373921 697558 933267 937405 144482 496880 44369 938415 497243 626210 81083 391750 449848 908621 436724 75067 873538 87548 815960 542364 309111 420492 213119 825861 873321 806546 7274 817536 922857 908199 619877 152536 86683 115725 315578 248449 65613 957220 132832 564689 929284 962586 395464 546360 828561 739676 115587 444684 155262 906871 153722 48905 479409 625567 854252 715474 216700 600556 161515 117929 