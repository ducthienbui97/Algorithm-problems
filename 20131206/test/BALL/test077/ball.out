4101818465288
